File: my-folder/0001-two-sum/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int>mp;
        unordered_set<int>st;
        vector<int>ans;
        for(int i=0;i<nums.size();i++){
            if(st.find(target-nums[i])!=st.end()) return {mp[target-nums[i]],i};
            st.insert(nums[i]);
            mp[nums[i]]=i;
        }
        return ans;
    }
};

File: my-folder/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums, target):
        mp = {}
        b = []
        for i in range(len(nums)):
            if (target - nums[i]) in mp:
                b.append(i)
                b.append(mp[target - nums[i]])
                return b

            mp[nums[i]] = i
        return b
